Italian Authentic Sources' Onboarding and EAA design Handbook (#1063)
italia/eidas-it-wallet-docs: manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md

# Manuale operativo Titolare di Fonte Autentica IT-Wallet

### Guida operativa per la messa a disposizione dei dati per l'emissione di Attestati Elettronici di Attributi sulle soluzioni IT-Wallet (include Casi d'uso e Assistenza)

**Versione** 1.0.0

> > **Nota introduttiva** — Il presente documento non ridefinisce quanto già definito all'interno delle Specifiche Tecniche. Qualora dovessero emergere interpretazioni diverse tra il manuale e le Specifiche Tecniche, il testo di queste ultime rappresenta la fonte normativa alla quale i manuali, come quello presente, devono attenersi.

Reviewer: ```suggestion
> **Nota introduttiva** — Il presente documento non ridefinisce quanto già definito all'interno delle Specifiche Tecniche. Qualora dovessero emergere interpretazioni diverse tra il manuale e le Specifiche Tecniche, il testo di queste ultime rappresenta la fonte normativa alla quale gli Enti devono attenersi.
```

## Indice dei contenuti

- [Introduzione e contesto](#Introduzione-e-contesto)
- [Scopo e ambito di applicazione](#Scopo-e-ambito-di-applicazione)
- [Ruoli e responsabilità del Titolare di Fonte Autentica](#Ruoli-e-responsabilità-del-Titolare-di-Fonte-Autentica)
- [Come diventare Titolare di Fonte Autentica](#Come-diventare-Titolare-di-Fonte-Autentica)

>   - [Step 1 | Progettazione dell'e-service e dei dati](#step-1--progettazione-dell-e-service-e-dei-dati) 

Reviewer: ```suggestion
  - [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa) 
```

to better represent all the activities proposed in step 1

Reviewer: @peppelinux 
Possible alternative to express both "dataset" and "info related to discovery, issuance, usage and management" concepts: "Step 1 | Progettazione dati e caratteristiche EAA"

cc: @RosaliaGaleano

  - [Step 2 | Pubblicazione in collaudo](#step-2--pubblicazione-in-collaudo) 
  - [Step 3 | Test in collaudo](#step-3--test-in-collaudo)
  - [Step 4 | Pubblicazione in produzione](#step-4--pubblicazione-in-produzione) 
  - [Step 5 | Test in produzione](#step-5--test-in-produzione)
  - [Step 6 | Pianificazione rilascio EAA](#step-6--pianificazione-rilascio-eaa)
  - [Step 7 | Manutenzione e assistenza](#step-7--manutenzione-e-assistenza)

> - [Documenti da compilare](#Documenti-da-compilare)

Reviewer: ```suggestion
- [File da compilare](#file-da-compilare)
```
to better refer to json file

> - [Appendice A – Casi d'uso EAA](#appendice-a--casi-duso-eaa)

Reviewer: ```suggestion
- [Appendice A – Casi d'uso](#appendice-a--casi-duso)
- [Appendice B – Data Model](#appendice-b--data-model)
- [Appendice C – Mappatura errori](#appendice-c--mappatura-errori)
- [Appendice D – Mappatura stati](#appendice-c--mappatura-stati)
```

We propose to link each section of the json file to a specific appendix

> - [Appendice B – Assistenza EAA](#appendice-b--assistenza-eaa)

Reviewer: ```suggestion
- [Appendice E – Assistenza](#appendice-e--assistenza)
```

> - [Appendice C – Guida compilazione e-service](#appendice-c--guida-compilazione-e-service)

Reviewer: ```suggestion
```

- [Template PDND Data Model](#Template-PDND-Data-Model)

## Introduzione e contesto

Il presente manuale rappresenta una **guida per gli Enti pubblici e privati che sono interessati a svolgere il ruolo di Titolari di Fonte Autentica nel Sistema IT-Wallet** e a **esporre dati per l'emissione di Attestati Elettronici di Attributi (Electronic Attestation of Attributes - EAA) nelle soluzioni IT-Wallet**.

> I **Destinatari** del presente manuale sono gli **implementatori di Fonte Autentica**. Si precisa che le Fonti Autentiche **non emettono** Attestati Elettronici di Attributi (EAA): forniscono i dati tramite un **e-service PDND** al Fornitore di Attestati Elettronici (es. IPZS), che procede all'emissione degli EAA verso gli utenti.

Reviewer: ```suggestion
```

reduntant with content described below

L'Ente che fornisce i dati, in quanto Titolare di Fonte Autentica, è l'unico soggetto che può definire mezzi e finalità di uso degli stessi. L'Ente rimarrà sempre il "proprietario" del dato e sarà responsabile del ciclo di vita del dato quindi di eventuali modifiche o cambiamenti di stato. Il Sistema IT-Wallet è concepito in modo che l'Ente possa continuare a gestire il dato in modo autonomo e conforme alle proprie politiche e alle normative vigenti. 

Di seguito rappresentato il ruolo del Titolare di Fonte Autentica e delle altre entità nel Sistema IT-Wallet. 

![Ruolo del Titolare di Fonte Autentica nel Sistema IT-Wallet](https://github.com/user-attachments/assets/e23b1cd0-b547-408e-8354-58131e0977b4)

*Figura 1: Ruolo del Titolare di Fonte Autentica nel Sistema IT-Wallet*

Di seguito il ruolo del Titolare di Fonte Autentica nel contesto del flusso di richiesta ed emissione di un EAA. 

![Flusso di richiesta ed emissione di un EAA nel Sistema IT-Wallet](https://github.com/user-attachments/assets/9223e9fa-9830-4e1d-b1ac-6354554ef2ce)

*Figura 2: Flusso di richiesta ed emissione di un EAA nel Sistema IT-Wallet* 

> > **NB:** Nel Sistema IT-Wallet IPZS (Istituto Poligrafico e Zecca dello Stato) è l'unico Fornitore di Attestati Elettronici di Attributi di Interesse Pubblico, coerentemente con la normativa. PagoPA S.p.A. è l'unico fornitore di soluzione pubblica di IT-Wallet, ospitata all'interno dell'app IO, l'app dei servizi pubblici per la PA.

Reviewer: ```suggestion
> **NB:** Nel Sistema IT-Wallet, coerentemente con la normativa, IPZS (Istituto Poligrafico e Zecca dello Stato) è l'unico Fornitore di Attestati Elettronici di Attributi di Interesse Pubblico e PagoPA S.p.A. è l'unico fornitore di soluzione pubblica di IT-Wallet, ospitata all'interno dell'app IO, l'app dei servizi pubblici.
```

## Come diventare Titolare di Fonte Autentica

> Per rivestire il ruolo di Titolare di Fonte Autentica, e quindi **implementare e pubblicare un e-service PDND** che metta a disposizione i propri dati al Fornitore di Attestati Elettronici (il quale emetterà gli EAA agli utenti), ciascun Ente interessato deve attenersi al seguente processo di onboarding tecnico, da considerarsi valido fino alla pubblicazione del Regolamento IT-Wallet e alla disponibilità di:

Reviewer: ```suggestion
Per rivestire il ruolo di Titolare di Fonte Autentica, ciascun Ente interessato deve attenersi al seguente processo di onboarding tecnico, da considerarsi valido fino alla pubblicazione del Regolamento IT-Wallet e alla disponibilità di:
```
reduntant with preview section dedicated to AS and Issuer roles

- Portale di Onboarding
- Registro delle Fonti Autentiche
- Catalogo degli Attestati Elettronici

In particolare, il processo prevede i seguenti step: 

> - **Step 1 | Progettazione dell'e-service e dei dati**: l'Ente approfondisce le Specifiche Tecniche del Sistema IT-Wallet e definisce il data model dell'e-service, i casi d'uso, le modalità di gestione di errori e stati dei dati. Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
- **Step 1 | Progettazione caratteristiche EAA**: l'Ente approfondisce le Specifiche Tecniche del Sistema IT-Wallet e definisce le caratteristiche del Attestato Elettronico di Attributi relativo al proprio dataset, in relazione alle modalità di scoperta e ottenimento dell’EAA, i casi d’uso, il data model e le modalità di gestione di errori e stati. 
```

Editorial changes + Proposal with no external references, in order to maintain a simple and clear overview of the overall process. References to tech specifications are maintained in each step deep dive

Author: modificato, non usiamo termini tipo "il proprio EAA".

>   - [Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)

Reviewer: ```suggestion

```

>   - [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)

Reviewer: ```suggestion

```

>   - [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)

Reviewer: ```suggestion

```

>   [Vai allo Step 1](#step-1--progettazione-dell-e-service-e-dei-dati)

Reviewer: ```suggestion
  [Vai allo Step 1](#step-1--progettazione-caratteristiche-eaa)
```

> - **Step 2 | Pubblicazione in collaudo**: l'Ente effettua l'onboarding nella Piattaforma Digitale Nazionale Dati (PDND), se non lo ha già fatto, rilascia l'e-service in ambiente di collaudo su PDND, e attiva il relativo servizio Signal Hub in ambiente di collaudo. Infine, l'Ente notifica al Fornitore di Attestati Elettronici di Attributi, configurato come fruitore dell'e-service, l'avvenuta pubblicazione. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:

Reviewer: ```suggestion
- **Step 2 | Pubblicazione in collaudo**: l'Ente effettua l'onboarding nella Piattaforma Digitale Nazionale Dati (PDND), se non lo ha già fatto, rilascia l'e-service in ambiente di collaudo su PDND, e attiva il relativo servizio Signal Hub in ambiente di collaudo per la gestione del ciclo di vita dell’EAA nel tempo. Infine, l'Ente notifica al Fornitore di Attestati Elettronici di Attributi, configurato come fruitore dell'e-service, l'avvenuta pubblicazione. 
```

>   - [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)

Reviewer: ```suggestion
```

>   - [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)

Reviewer: ```suggestion
```

>   - [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)

Reviewer: ```suggestion
```

  [Vai allo Step 2](#step-2--pubblicazione-in-collaudo)

> - **Step 3 | Test in collaudo**: l'Ente, in ambiente di collaudo PDND, può eseguire i test di integrazione dell'e-service e di Signal Hub con il Fornitore di Attestati Elettronici di Attributi indicato come fruitore e, se ritenuto necessario, con il Fornitore di Wallet. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:

Reviewer: ```suggestion
- **Step 3 | Test in collaudo**: l'Ente, in ambiente di collaudo PDND, può eseguire i test di integrazione dell'e-service e di Signal Hub con il Fornitore di Attestati Elettronici di Attributi indicato come fruitore e, se ritenuto necessario, con il Fornitore di Wallet, anche gli aspetti relativi alla UX/UI dell’EAA. 
```

>   - [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)

Reviewer: ```suggestion

```

>   - [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)

Reviewer: ```suggestion

```

  [Vai allo Step 3](#step-3--test-in-collaudo)

> - **Step 4 | Pubblicazione in produzione**: l'Ente rilascia l'e-service in ambiente di produzione su PDND e attiva il relativo servizio Signal Hub in produzione. Infine, l'Ente notifica al Fornitore di Attestati Elettronici di Attributi. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:

Reviewer: ```suggestion
- **Step 4 | Pubblicazione in produzione**: l'Ente rilascia l'e-service in ambiente di produzione su PDND e attiva il relativo servizio Signal Hub in produzione, al fine di supportare una corretta gestione del ciclo di vita dell’EAA. Infine, l'Ente notifica al Fornitore di Attestati Elettronici di Attributi l'avvenuta pubblicazione.
```

>   - [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)

Reviewer: ```suggestion

```

>   - [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)

Reviewer: ```suggestion

```

  [Vai allo Step 4](#step-4--pubblicazione-in-produzione)
- **Step 5 | Test in produzione**: l'Ente, in ambiente di produzione, esegue i test di integrazione, di carico e long run dell'e-service. Per i dettagli implementativi, consultare le Specifiche Tecniche italiane:

>   - [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)

Reviewer: ```suggestion

```

  [Vai allo Step 5](#step-5--test-in-produzione)

> - **Step 6 | Pianificazione rilascio EAA**: a valle del buon esito dei test in collaudo e in produzione, l'Ente concorda con il Fornitore di Attestati Elettronici di Attributi la data di rilascio. Per la comunicazione e il rilascio, consultare le Specifiche Tecniche italiane:

Reviewer: ```suggestion
- **Step 6 | Pianificazione rilascio EAA**: a valle del buon esito dei test in collaudo e in produzione, l'Ente concorda con il Fornitore di Attestati Elettronici di Attributi la data di rilascio, per l'ottenimento dell’EAA da parte degli utenti. Inoltre, l’Ente può valutare attività di comunicazione, in sinergia con gli altri attori interessati.
```

>   - [Brand Identity](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/brand-identity.html)

Reviewer: ```suggestion

```

  [Vai allo Step 6](#step-6--pianificazione-rilascio-eaa)

> - **Step 7 | Manutenzione e assistenza**: l'Ente effettua attività di gestione e manutenzione dell'e-service e contribuisce alla risoluzione di problematiche. Per il modello di assistenza, consultare le Specifiche Tecniche italiane:

Reviewer: ```suggestion
- **Step 7 | Manutenzione e assistenza**: l'Ente effettua attività di gestione e manutenzione dell'e-service e contribuisce alla risoluzione di problematiche e segnalazioni, per le tematiche e i processi di competenza, secondo il modello di assistenza del Sistema IT-Wallet. 
```

>   - [Assistenza utente](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente)

Reviewer: ```suggestion

```

  [Vai allo Step 7](#step-7--manutenzione-e-assistenza)

> ## Step 1 | Progettazione dell'e-service e dei dati

Reviewer: ```suggestion
## Step 1 | Progettazione caratteristiche EAA
```

> Questo step ha l'obiettivo di definire, in qualità di Fonte Autentica, come guidare gli utenti all'ottenimento degli EAA prodotti mediante i propri dati, quali dati fornire e come gestirli nel tempo. Le attività previste in questa fase sono di piena responsabilità della Fonte Autentica e contribuiscono all'esperienza utente (es. modalità di discovery, casi d'uso, qualità dei dati, assistenza, condizioni di validità dell'EAA prodotto). In questo step, l'Ente interessato deve:

Reviewer: ```suggestion
Questo step ha l'obiettivo di definire l'esperienza utente di scoperta, ottenimento, utilizzo e gestione dell’Attestato Elettronico di Attributi relativo al dataset rilasciato dalla Fonte Autentica. Le attività previste in questa fase riguardano l'Ente nei limiti delle proprie responsabilità, contribuendo all'esperienza utente e più in generale alla qualità del servizio finale (es. modalità di discovery, casi d'uso, qualità dei dati, assistenza, condizioni di validità dell'EAA). In questo step, l'Ente interessato deve: 
```

Author: utilizzo e gestione dell'EAA non sono "di piena responsabilità dell’Ente"

modificato

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion

```

Already described in the following sub section "Approfondire le Specifiche Tecniche del Sistema IT-Wallet"

> - [Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)

Reviewer: ```suggestion

```

> - [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)

Reviewer: ```suggestion

```

> - [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)

Reviewer: ```suggestion

```

> - [Design UX](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html)

Reviewer: ```suggestion

```

### **Approfondire le Specifiche Tecniche del Sistema IT-Wallet**

La versione corrente delle Specifiche Tecniche è disponibile a [questo link](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/). Si raccomanda l'approfondimento, in particolare, delle sezioni: 

- [Design dell'Esperienza Utente](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html) 
- [Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)  
- [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html) 
- [Algoritmi Crittografici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/algorithms.html) 
- [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html) 
- [Soluzione del Fornitore di Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-issuer-solution.html)
- [Gestione degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/digital-credential-management.html)

> ### **Definire come guidare l'utente all'ottenimento dell'EAA**

Reviewer: ```suggestion
### **Definire come l'utente può ottenere l'EAA**
```

> Il Sistema IT-Wallet consente agli utenti di scoprire e ottenere gli EAA attraverso diverse modalità. La Fonte Autentica, fornendo i dati da cui vengono prodotti gli EAA, può **orientare gli utenti** verso l'ottenimento di tali attestati. In particolare: 

Reviewer: ```suggestion
Il Sistema IT-Wallet consente agli utenti di ottenere gli EAA attraverso diverse modalità. In particolare: 
```

> - l'utente può avviare il flusso di ottenimento dell'EAA prodotto con i propri dati: 

Reviewer: ```suggestion
- l'utente può avviare il flusso di ottenimento dell'EAA: 
```

to avoid possible misunderstaning about user'data

  - **attraverso una sezione "catalogo" della soluzione IT-Wallet**, soluzione indicata per EAA di interesse nazionale o rivolti a un'ampia percentuale di popolazione (es. Tessera Sanitaria, Patente di guida, etc.). Per approfondimenti vai alle Specifiche Tecniche:
    - [Ottenimento dal Catalogo dell'Istanza del Wallet](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)
    - [Issuance Flow](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-issuance-low-level.html#issuance-flow)
  - **attraverso uno dei touchpoint della Fonte Autentica** (es. sito web), soluzione indicata per EAA di interesse locale o rivolti a un pubblico specifico (es. certificati, prenotazioni, etc.). La Fonte Autentica può così **guidare l'utente** verso l'ottenimento dell'EAA tramite Credential Offer. Questo flusso può affiancarsi o sostituire il precedente a seconda del tipo di EAA. Per approfondimento vai alle Specifiche Tecniche:

>     - [Ottenimento dal Touchpoint della Fonte Autentica](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-touchpoint-della-fonte-autentica)

Reviewer: ```suggestion
    - [Ottenimento dal Touchpoint del Titolare della Fonte Autentica](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-touchpoint-della-fonte-autentica)
```

    - [Credential Offer](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-issuance-low-level.html#flusso-credential-offer)

> - la Fonte Autentica può rispondere alla richiesta di emissione (tramite e-service) in: 

Reviewer: ```suggestion
- il Titolare di Fonte Autentica può rispondere alla richiesta di emissione (tramite e-service) in: 
```

  - **modalità sincrona**, che consente l'ottenimento immediato dell'EAA da parte dell'utente e si configura come l'opzione preferibile;
  - **modalità differita**, che consente all'utente di ottenere l'EAA non contestualmente al momento della richiesta e si configura come l'opzione non preferibile.

> È necessario quindi che la Fonte Autentica definisca a monte le modalità con cui gli utenti saranno guidati all'ottenimento degli EAA prodotti con i propri dati, sulla base di determinati parametri: a chi si rivolge l'EAA (a tutta la popolazione o solo a una nicchia di persone?); cosa deve fare l'utente per ottenere l'EAA (è necessario essere in possesso di specifici prerequisiti? deve effettuare un processo di richiesta/adesione/pagamento? etc.); tramite quali canali l'utente potrà richiedere l'EAA e quando potrà ottenerlo (contestualmente o non contestualmente alla richiesta).  

Reviewer: ```suggestion
È necessario quindi che l'Ente definisca a monte le modalità di ottenimento dell‘EAA reso disponibile grazie ai propri dati, sulla base di determinati parametri: a chi si rivolge l'EAA (a tutta la popolazione o solo a una nicchia di persone?); cosa deve fare l'utente per ottenere l'EAA (è necessario essere in possesso di specifici prerequisiti? deve effettuare un processo di richiesta/adesione/pagamento? etc.); tramite quali canali l'utente potrà richiedere l'EAA e quando potrà ottenerlo (contestualmente o non contestualmente alla richiesta). 
```

> ### **Definire i casi d'uso dell'EAA prodotto con i propri dati**

Reviewer: ```suggestion
### **Definire i casi di utilizzo dell'EAA**
```

> Gli EAA prodotti con i dati forniti dalla Fonte Autentica possono essere utilizzati dall'utente in diversi modi. È utile che la Fonte Autentica definisca i casi d'uso previsti per orientare la progettazione complessiva. IT-Wallet consente all'utente di utilizzare i propri EAA: 

Reviewer: ```suggestion
Il Sistema IT-Wallet consente agli utenti di utilizzare gli EAA in diverse modalità:
```

  - [Presentazione da remoto](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#presentazione-da-remoto)
  - [Flusso remoto](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/remote-flow.html)
- **in prossimità** (in presenza), tramite flussi **supervisionati**, ovvero mostrando un QR code che il Verificatore potrà verificare tramite apposita app di verifica o **non supervisionati**, ovvero tramite strumenti di verifica automatica (tornelli, totem, etc.) con l'utilizzo di tecnologie sicure come il Bluetooth o NFC. Per approfondimenti vai alle Specifiche Tecniche:
  - [Presentazione in prossimità](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#presentazione-in-prossimita)
  - [Flusso di prossimità](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/proximity-flow.html)

> La definizione dei casi d'uso da parte della Fonte Autentica è fondamentale per: 

Reviewer: ```suggestion
La definizione dei casi d'uso da parte dell'Ente è fondamentale per: 
```

- **progettare un'esperienza d'uso che apporti valore reale** sia a cittadini che ai verificatori; è utile definire a monte quali potranno essere le occasioni d'uso dell'EAA prodotto con i propri dati, a partire dall'analisi dell'esperienza attuale di fruizione del corrispettivo documento fisico, se esistente (es. si pensi alla modalità di presentazione del codice a barre per l'uso della Tessera Sanitaria o del QR code per la verifica della Carta Europea della Disabilità); 
- **orientare il tipo di formato** con cui il Fornitore di Attestati Elettronici emetterà l'EAA (SD-JWT-VC per scenari in remoto e mdoc-CBOR per scenari in prossimità).

> A tal fine, la Fonte Autentica deve compilare il questionario Casi d'uso EAA in formato JSON, utilizzando il [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json) (vedi [Appendice A](#appendice-a--casi-duso-eaa)). Valida il file JSON con lo strumento indicato nel [documento di validazione](validazione-json-schema-linter.md) prima della sottomissione. 

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione `casi_d_uso` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Per riferimenti e istruzioni di compilazione vedi [Appendice A](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-a--casi-duso).
```

We propose to add an additional sub-Step dedicated to file validation, at the end of Step 1, in order to invite the user to validate the entire file once at the end of step 1

> ### **Definire il Data Model dei dati forniti**

Reviewer: ```suggestion
### **Definire il Data Model**
```

> Gli EAA sono attestati digitali costruiti sui dati forniti dalla Fonte Autentica tramite l'e-service. Il Fornitore di Attestati Elettronici produce l'EAA a partire da tali dati.  

Reviewer: ```suggestion
Il Sistema IT-Wallet consente all’utente di ottenere in formato digitale i propri documenti, titoli e certificati sotto forma di EAA. Gli EAA sono rilasciati dal Fornitore di Attestati Elettronici di Attributi sulla base dei dati forniti dalla Fonte Autentica tramite l'e-service.
```

> La Fonte Autentica deve definire quali dati fornirà e in quale ordine, affinché l'EAA prodotto risulti adeguato all'utilizzo in versione digitale e subito comprensibile all'utente. 

Reviewer: ```suggestion
L’Ente deve quindi definire quali dati fornirà e in quale ordine, affinché l'EAA prodotto risulti adeguato all'utilizzo da parte dell’utente. 
```

> A tal fine, l'Ente deve compilare la sezione **e_service.response.data_model** del [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json), dichiarando i dettagli sui dati che verranno messi a disposizione (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.).  

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione `e_service.response.data_model ` del file [Progettazione caratteristiche EAA](template-form-onboarding-fonte-autentica.json) così da definire i dettagli sui dati che verranno messi a disposizione (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice B](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-b--data-model).
```

> Nella sezione [Template PDND Data Model](#Template-PDND-Data-Model) sono riportati i riferimenti dei template Data Model pubblicati su PDND relativi ad alcune tipologie di attestati emessi in IT-Wallet; l'Ente allinea il proprio data model (sezione **e_service.response.data_model**) a tali template per garantire che i dati forniti siano conformi alla struttura dell'attestato che verrà emesso dal Fornitore di Attestati.

Reviewer: ```suggestion
Nella sezione [Template PDND Data Model](#Template-PDND-Data-Model) sono riportati i riferimenti dei template Data Model pubblicati su PDND relativi ad alcune tipologie di attestati emessi in IT-Wallet; l'Ente allinea il proprio data model (sezione `e_service.response.data_model`) a tali template per garantire che i dati forniti siano conformi alla struttura dell'Attestato che verrà emesso dal Fornitore di Attestati di Attributi.
```

In conclusione, un'adeguata definizione del Data Model pone le basi per una corretta implementazione dell'e-service da pubblicare su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)) ma è altresì importante considerare e rispettare i seguenti requisiti tecnici: 

- **Identificativo utente**: Qualora fosse necessario identificare l'utente, il Codice Fiscale (CF) rappresenta l'identificativo univoco da utilizzare per le chiamate all'e-service; 
- **Completezza base di dati**: Ogni e-service pubblicato su PDND dovrà esporre un set di dati completo nel contenuto e negli attributi. È ammessa la pubblicazione di basi dati parziali relative a periodi temporali limitati.

### **Definire le casistiche di errore**

> L'e-service messo a disposizione dall'Ente deve prevedere e gestire specifiche situazioni di errore che possono verificarsi nella fase di recupero dei dati da parte del Fornitore di Attestati.

Reviewer: ```suggestion
L'e-service messo a disposizione dall'Ente deve prevedere e gestire specifiche situazioni di errore che possono verificarsi nella fase di recupero dei dati da parte del Fornitore di Attestati di Attributi.
```

> A tal fine, l'Ente deve compilare la sezione **e_service.response.mappatura_errori** del [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire per garantire una **corretta informazione all'utente in caso di errori** durante l'ottenimento dell'EAA. 

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `e_service.response.mappatura_errori `del file [Progettazione caratteristiche EAA](template-form-onboarding-fonte-autentica.json). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-c--mappatura-errori).
```

### **Definire la gestione degli stati del ciclo di vita**

> Gli stati che l'Ente comunica tramite Signal Hub influenzano il ciclo di vita degli EAA prodotti dai propri dati.  

Reviewer: ```suggestion
Il Sistema IT-Wallet supporta dei meccanismi per l’aggiornamento dello stato e la gestione del ciclo di vita dell’EAA. Gli stati che l'Ente comunica tramite Signal Hub determinano il ciclo di vita degli EAA prodotti dai propri dati.  
```

- **Scaduto**: EAA con data di scadenza amministrativa superata. La scadenza amministrativa può essere definita dal Titolare di Fonte Autentica e, se ritenuta utile o necessaria, deve essere inclusa come attributo all'interno del data model dell'EAA per garantire messaggi informativi all'interno della soluzione IT-Wallet (es. La tua Patente scade tra 30 giorni);
- **Da aggiornare**: EAA con data di scadenza tecnica superata. La scadenza tecnica è definita del Fornitore di Attestati Elettronici di Attributi ed è impostata generalmente a 1 anno o comunque a un valore inferiore o uguale alla data di scadenza amministrativa. Tale scadenza ha l'obiettivo di richiedere un'azione di riemissione esplicita all'utente e mitigare rischi di sicurezza.

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

> A tal fine, l'Ente deve compilare la sezione **e_service.response.stati** del [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json) per definire i dettagli sugli stati relativi al ciclo di vita dell'EAA, ovvero dichiarare la condizione di applicabilità dei tre stati sopracitati e l'eventuale relativo messaggio informativo da esporre all'utente.

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `mappatura-stati` del file [Progettazione caratteristiche EAA](e_service.response.data_model) per definire i dettagli sugli stati relativi al ciclo di vita dell'EAA, ovvero dichiarare la condizione di applicabilità dei tre stati sopracitati e l'eventuale relativo messaggio informativo da esporre all'utente.
Per riferimenti e istruzioni di compilazione vedi [Appendice D](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-d--mappatura-stati).
```

**Nota**: 
Per ottimizzare l'esperienza d'uso dell'IT-Wallet pubblico, il Titolare di Fonte Autentica può anche valutare l'**integrazione con app IO per l'invio di messaggi informativi al cittadino**, quali ad esempio:

- comunicare il cambio di stato del documento (come, ad esempio, un reminder sulla scadenza)
- informarlo che il nuovo documento è pronto per essere ritirato
- novità legate ai servizi offerti (opzionale ma consigliato).

### **Definire i contenuti per l'informazione e l'assistenza all'utente**

L'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi in app, e fornendo i recapiti necessari per la gestione dell'assistenza agli utenti.  

> A tal fine, l'Ente deve compilare la sezione **assistenza** nel [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json) (vedi [Appendice B](#appendice-b--assistenza-eaa)). 

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `assistenza` del file [Progettazione caratteristiche EAA](template-form-onboarding-fonte-autentica.json). Per riferimenti e istruzioni di compilazione vedi [Appendice E](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#appendice-e--assistenza).
```

> ### **Definire le caratteristiche grafiche dell'EAA prodotto dai propri dati**

Reviewer: ```suggestion
### **Predisporre gli elementi necessari per la rappresentazione grafica dell'EAA**
```

> La rappresentazione grafica dell'EAA nell'IT-Wallet può dipendere, per specifici aspetti, da parametri definiti nelle Specifiche Tecniche, sezione [Focus sugli Attestati Elettronici di Attributi](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#focus-sugli-attestati-elettronici-di-attributi). 

Reviewer: ```suggestion
Il Sistema IT-Wallet consente ai Titolari di Fonte Autentica di contribuire alla resa grafica degli EAA prodotti a partire dai propri dati. La rappresentazione visiva di un EAA all’interno di un IT-Wallet può dipendere quindi, per specifici aspetti, da parametri definiti nelle Specifiche Tecniche, sezione [Focus sugli Attestati Elettronici di Attributi](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#focus-sugli-attestati-elettronici-di-attributi). 
```

> L'Ente interessato a personalizzare la resa grafica dell'EAA prodotto dai propri dati deve trasmettere i propri materiali come documentazione allegata all'e-service su PDND.  

Reviewer: ```suggestion
L'Ente interessato a personalizzare la resa grafica dell'EAA prodotto dai propri dati deve trasmettere i propri materiali come documentazione allegata all'e-service su PDND (vedi [Step 2](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#step-2--pubblicazione-in-collaudo)).
```

## Step 2 | Pubblicazione in collaudo

> Questo step ha l'obiettivo di rilasciare in collaudo su PDND l'e-service che espone i dati per la produzione degli EAA e di attivare il relativo servizio Signal Hub per la gestione dei dati nel tempo. Inoltre, l'Ente può rilasciare in collaudo il Credential Offer se ritenuto utile per guidare gli utenti all'ottenimento degli EAA prodotti con i propri dati.

Reviewer: ```suggestion
Questo step ha l'obiettivo di rilasciare in collaudo su PDND l'e-service che espone i dati per la produzione degli EAA e di attivare il relativo servizio Signal Hub per la gestione dei dati nel tempo. Inoltre, l'Ente può rilasciare in collaudo il Credential Offer se ritenuto utile durate lo [Step 1](https://github.com/italia/eid-wallet-it-docs/blob/9f6806493b550eb60cdcff8f95d5422c9a05a10a/manuals/it/authentic-sources/manuale-titolare-fonte-autentica.md#step-1--progettazione-caratteristiche-eaa)).
```

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:
```

- [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)
- [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)
- [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)
- [Template e-service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd-template.html)

> 

Reviewer: ```suggestion
In questo step, l'Ente interessato deve: 
```

### **Aderire alla Piattaforma Digitale Nazionale Dati (PDND)**

> Per effettuare l'onboarding alla PDND, qualora l'Ente ancora non abbia aderito, consultare la [Guida all'adesione](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/guida-alladesione). 

Reviewer: ```suggestion
Per effettuare l'onboarding alla PDND, qualora l'Ente ancora non abbia aderito, consultare il [Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/guida-alladesione). 
```

### **Pubblicare l'e-service su PDND in collaudo**

> L'Ente deve sviluppare e rilasciare in collaudo un e-service coerente con il Data Model precedentemente definito, in linea con le informazioni presenti nella Guida all'adesione di PDND e con le Specifiche Tecniche italiane. Per i dettagli operativi e le specifiche tecniche, consultare:

Reviewer: ```suggestion
L'Ente deve sviluppare e rilasciare in collaudo un e-service coerente con il Data Model precedentemente definito, in linea con le informazioni presenti nel [Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/guida-alladesione) e con le Specifiche Tecniche italiane. Per i dettagli operativi e le Specifiche Tecniche, consultare:
```

> - [sezione e-service](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/e-service)

Reviewer: ```suggestion
- [sezione e-service del Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/e-service)
```

> - [Titolare di Fonte Autentica](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)

Reviewer: ```suggestion
- [sezione Titolare di Fonte Autentica delle Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)
```

> Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare i seguenti documenti, precedentemente compilati:

Reviewer: ```suggestion
Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare il file [Progettazione caratteristiche EAA](template-form-onboarding-fonte-autentica.json) precedentemente compilato in tutte le sue parti e e validato (sezioni `casi_d_uso`, `data_model`, `mappatura_errori`, `mappatura_stati` e `assistenza`). 
```

> - [Template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json) — include Casi d'uso, Assistenza e sezione **e_service** (data model, mappatura errori, stati, lista nome campo). Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione dell'e-service, se possibile con abilitazione automatica.

Reviewer: ```suggestion
Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione dell'e-service, se possibile con abilitazione automatica.
```

> Si consiglia di nominare l'e-service in "Creazione EAA [Nome / Nome tipologia EAA] – IT-Wallet" (es. "Creazione EAA Patente di guida – IT-Wallet" oppure "Creazione EAA Titoli di studio – IT-Wallet") e di predisporre una descrizione in linea con la [Guida alla nomenclatura degli e-service PDND](https://italia.github.io/pdnd-guida-nomenclatura-eservice/index.html) referenziata nella Guida all'adesione PDND. 

Reviewer: ```suggestion
Si consiglia di nominare l'e-service in "Creazione EAA [Nome / Nome tipologia EAA] – IT-Wallet" (es. "Creazione EAA Patente di guida – IT-Wallet" oppure "Creazione EAA Titoli di studio – IT-Wallet") e di predisporre una descrizione in linea con il [Manuale Operativo PDND Interoperabilità](https://italia.github.io/pdnd-guida-nomenclatura-eservice/index.html) referenziata nella Guida all'adesione PDND. 
```

### **Attivare il servizio Signal Hub in collaudo**

> L'Ente deve attivare in collaudo il servizio [Signal Hub](https://developer.pagopa.it/pdnd-interoperabilita/guides/manuale-operativo-pdnd-interoperabilita/v1.0/riferimenti-tecnici/signal-hub) di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione **e_service.response.stati** del [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json). 

Reviewer: ```suggestion
L'Ente deve attivare in collaudo il servizio [Signal Hub](https://developer.pagopa.it/pdnd-interoperabilita/guides/manuale-operativo-pdnd-interoperabilita/v1.0/riferimenti-tecnici/signal-hub) di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura_stati` del file [Progettazione caratteristiche EAA](template-form-onboarding-fonte-autentica.json).  
```

### **Sviluppare il Credential Offer (opzionale)**

L'Ente, se ritenuto necessario e/o utile per ingaggiare l'utente nel flusso di ottenimento dell'EAA, deve provvedere in questo step agli sviluppi e all'integrazione del Credential Offer all'interno delle proprie soluzioni.  

### **Notificare l'avvenuto rilascio in collaudo**

> L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in collaudo su PDND, dei servizi di Signal Hub in collaudo e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo XX.

Reviewer: ```suggestion
L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in collaudo su PDND, dei servizi di Signal Hub in collaudo e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo identitadigitale@pec.ipzs.it.
```

## Step 3 | Test in collaudo

> Questo step, suggerito ma non vincolante per le fasi successive, ha l'obiettivo di eseguire i test propedeutici al rilascio in produzione. In caso di disponibilità da parte del Fornitore di Attestati Elettronici di Attributi, l'Ente può:

Reviewer: ```suggestion
Questo step, suggerito ma non vincolante per le fasi successive, ha l'obiettivo di eseguire i test propedeutici al rilascio in produzione. 
```

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:
```

- [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)
- [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)

> 

Reviewer: ```suggestion
In questo step, in caso di disponibilità da parte del Fornitore di Attestati Elettronici di Attributi, l'Ente interessato può: 
```

### **Eseguire i test in collaudo**

L'Ente **può** testare in collaudo con il Fornitore di Attestati Elettronici di Attributi (e.g. IPZS, nel caso di Attestati Elettronici di interesse pubblico):

- la corretta erogazione dei dati tramite l'e-service in PDND
- la corretta fruizione del servizio Signal Hub di PDND per la gestione del ciclo di vita dell'EAA
- se previsto, il Credential Offer

Infine, se ritenuto necessario, l'Ente può verificare la resa grafica dell'EAA di interesse con il Fornitore di Wallet (e.g. PagoPA, nel caso di soluzione pubblica di IT-Wallet).

Una volta superati i test in collaudo, se eseguiti, l'Ente può proseguire con la fase successiva.

## Step 4 | Pubblicazione in produzione

> Questo step ha l'obiettivo di rilasciare in produzione l'e-service che espone i dati per la produzione degli EAA e di attivare il relativo servizio Signal Hub per la gestione dei dati nel tempo. Inoltre, l'Ente può rilasciare in produzione il Credential Offer se ritenuto utile per guidare gli utenti all'ottenimento degli EAA prodotti con i propri dati. A tale scopo, l'Ente deve:

Reviewer: ```suggestion
Questo step ha l'obiettivo di rilasciare in produzione l'e-service che espone i dati per la produzione degli EAA e di attivare il relativo servizio Signal Hub per la gestione dei dati nel tempo. Inoltre, l'Ente può rilasciare in produzione il Credential Offer se ritenuto utile per guidare gli utenti all'ottenimento degli EAA prodotti con i propri dati. 
```

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:
```

- [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html)
- [Signal Hub](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html)

> 

Reviewer: ```suggestion
In questo step, l'Ente interessato deve:
```

### **Pubblicare l'e-service su PDND in produzione**

L'Ente deve rilasciare l'e-service su PDND in produzione. Nel caso di EAA di interesse pubblico, l'Ente deve abilitare IPZS alla fruizione del servizio, se possibile con abilitazione automatica. 

### **Attivare il servizio Signal Hub in produzione**

> L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione **e_service.response.stati** del [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json). 

Reviewer: ```suggestion
L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura_stati` del file [Progettazione caratteristiche EAA](template-form-onboarding-fonte-autentica.json).  
```

### **Portare in produzione il Credential Offer (opzionale)**

L'Ente, se precedentemente sviluppato in collaudo, deve provvedere in questo step al rilascio in produzione del Credential Offer all'interno delle proprie soluzioni.  

### **Notificare l'avvenuto rilascio in produzione**

> L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in produzione su PDND, dei servizi di Signal Hub in produzione e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo XX e, in caso di EAA disponibile nella soluzione pubblica di IT-Wallet, l'Ente deve notificare PagoPA inviando una mail all'indirizzo XX. 

Reviewer: ```suggestion
L'Ente deve notificare il Fornitore di Attestati di Attributi circa l'avvenuto rilascio dell'e-service in produzione su PDND, dei servizi di Signal Hub in produzione e, se previsto, del Credential Offer. In caso di EAA di interesse pubblico, l'Ente deve notificare IPZS inviando una mail all'indirizzo identitadigitale@pec.ipzs.it e, in caso di EAA disponibile nella soluzione pubblica di IT-Wallet, l'Ente deve notificare PagoPA inviando una mail all'indirizzo XX.
```

## Step 5 | Test in produzione

> Questo step ha l'obiettivo di eseguire i test necessari a un adeguato funzionamento di quanto rilasciato in produzione. A tale scopo, l'Ente deve:

Reviewer: ```suggestion
Questo step ha l'obiettivo di eseguire i test necessari a un adeguato funzionamento di quanto rilasciato in produzione. 
```

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:
```

- [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html)

> 

Reviewer: ```suggestion
In questo step, l'Ente interessato deve:
```

### **Effettuare i test in produzione**

L'Ente deve eseguire i test di tutte le componenti sviluppate (si consiglia un mese prima del go-live). In particolare:

- **Test di carico**: 300 richieste al secondo per almeno 60 minuti, con tempi di risposta inferiori a 500 millisecondi
- **Test di long run**: 150 richieste al secondo per almeno 12 ore consecutive, con tempi di risposta inferiori a 500 millisecondi

Una volta superati i test in produzione, l'Ente può proseguire con la fase successiva. 

## Step 6 | Pianificazione rilascio EAA

> Questo step ha l'obiettivo di pianificare e gestire le attività di rilascio agli utenti degli EAA prodotti con i propri dati. A tale scopo, l'Ente deve:

Reviewer: ```suggestion
Questo step ha l'obiettivo di pianificare e gestire le attività di rilascio agli utenti degli EAA prodotti con i propri dati. 
```

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:
```

- [Brand Identity](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/brand-identity.html)

> 

Reviewer: ```suggestion
In questo step, l'Ente interessato deve:
```

### **Pianificare il go-live**

A valle del buon esito dei test in collaudo e in produzione, l'Ente concorda con il Fornitore di Attestati Elettronici di Attributi e il Fornitore di Wallet la data di rilascio dell'EAA, quindi la possibilità di ottenimento dell'EAA da parte degli utenti. L'Ente dovrà in ogni caso effettuare la registrazione amministrativa, non appena disponibile, secondo quanto definito dal Regolamento IT-Wallet.

### **Valutare attività di comunicazione**

L'Ente può prevedere attività di comunicazione finalizzate a informare gli utenti della possibilità di ottenere e utilizzare l'EAA prodotto con i propri dati all'interno di IT-Wallet. Per approfondimento, vai alle Specifiche Tecniche, sezione [Brand Identity](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/brand-identity.html) del Sistema IT-Wallet. 

## Step 7 | Manutenzione e assistenza

> Una volta resi disponibili agli utenti gli EAA prodotti con i propri dati, la Fonte Autentica deve mantenere l'e-service e un ruolo attivo sia nella gestione dei dati (e del conseguente ciclo di vita degli EAA) che nell'assistenza agli utenti. In particolare, l'Ente deve:

Reviewer: ```suggestion
Una volta resi disponibili agli utenti gli EAA prodotti con i propri dati, la Fonte Autentica deve mantenere l'e-service e un ruolo attivo sia nella gestione dei dati (e del conseguente ciclo di vita degli EAA) che nell'assistenza agli utenti. 
```

> Per i dettagli implementativi, consultare le Specifiche Tecniche (IT), in particolare:

Reviewer: ```suggestion
Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:
```

- [Assistenza utente](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente)

> 

Reviewer: ```suggestion
In questo step, l'Ente interessato deve:
```

### **Garantire la gestione e manutenzione dell'e-service**

L'Ente deve garantire il corretto funzionamento dell'e-service nel tempo, programmare adeguate azioni di monitoraggio e aggiornamento se richieste, ad esempio, da cambiamenti normativi o procedurali (es. nuovi dati, stati, casistiche di errore, etc.). 

### **Gestire problematiche e fornire assistenza agli utenti**

> L'Ente deve garantire un costante aggiornamento delle informazioni riportate nella sezione **assistenza** del [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json), al fine di: 

Reviewer: ```suggestion
L'Ente deve garantire un costante aggiornamento delle informazioni riportate delle informazioni riportate nella sezione `assistenza` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) al fine di:
```

- **Contribuire alla risoluzione di bug** 
Il referente dell'ambito sistemistico e il referente dell'ambito applicativo, così come definito nella sezione `assistenza.referenti` del template, devono contribuire alla diagnosi congiunta delle segnalazioni ricevute da Fornitore di Attestati Elettronici di Attributi (IPZS, nel caso di EAA di interesse pubblico) e Fornitori di Wallet (PagoPA, nel caso della soluzione pubblica IT-Wallet) e relativa risoluzione, secondo quanto definito dal [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet. 
- **Garantire il supporto agli utenti** 

> Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione `assistenza` del template, devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.

Reviewer: ```suggestion
Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione `assistenza` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json), devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.
```

> ## Documenti da compilare

Reviewer: ```suggestion
## File da compilare
```

> Il template compila il **dataset e la configurazione della Fonte Autentica** (dati esposti, casi d'uso, assistenza, mappatura errori e stati): non si compila l'EAA, che viene emesso dal Fornitore di Attestati a partire dai dati forniti.

Reviewer: ```suggestion

```

> Di seguito l'elenco dei documenti da scaricare, compilare e condividere secondo le modalità definite durante:

Reviewer: ```suggestion

- scaricare il file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json)
- duplicare il file JSON per ciascuna EAA di interesse e rinomina ciascun file in "progettazione_ caratteristiche_eaa_[Nome Ente Titolare]_[Nome EAA]" (es. "progettazione_eaa_mim_titolo_di_studio.json"). 
- compilare ciascun file Json in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di dati esposti, casi d'uso, assistenza, mappatura errori e stati.  
- validare ciascun il file Json compilato utilizzando lo strumento indicato nel [file di validazione](https://github.com/italia/eid-wallet-it-docs/pull/1063/changes#diff-2337c8d8533155cad677214d3d5b723d77fa1f18fe26b99df6e9f6c91b4da9f5).

A compilazione e validazione conclusa, l’Ente può procedere con la sottomissione del file secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni di questa sezione secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

Il file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) contiene le seguenti sezioni: 

- Casi d’uso (istruzioni di compilazione in [Appendice A](#appendix-a)) 
- Data Model (istruzioni di compilazione in [Appendice B](#appendix-b)) 
- Mappatura errori (istruzioni di compilazione in [Appendice C](#appendix-c)) 
- Mappatura stati (istruzioni di compilazione in [Appendice D](#appendix-d)) 
- Assistenza (istruzioni di compilazione in [Appendice E](#appendix-e)) 
```

Since we have one file only, we suggest to anticipate here all the common operative steps to be followed for each JSON file sections.

> - [Step 1 - Progettazione dell'e-service e dei dati](#step-1--progettazione-dell-e-service-e-dei-dati)

Reviewer: ```suggestion
```

> - [Step 2 - Pubblicazione in collaudo](#step-2--pubblicazione-in-collaudo)

Reviewer: ```suggestion
```

> - [Template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json) — include Casi d'uso, Assistenza e **e_service** (data model, mappatura errori, stati, lista nome campo). Validare prima della sottomissione (vedi [Validazione JSON Schema e Linter](validazione-json-schema-linter.md)). Per le istruzioni dettagliate sulla sezione e_service, vedi [Appendice C](#appendice-c--guida-compilazione-e-service)

Reviewer: ```suggestion
```

## Appendice A – Casi d'uso EAA

> Il questionario Casi d'uso EAA si compila in formato JSON ed è incluso nel [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json). Le sezioni sotto riportate fungono da guida.

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `casi_d_uso` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire.
```

Need to harmonize labels for this section thoughtout the entire text (`casi_d_uso`  vs. `sezione_casi_uso`). Need to harmonize all the other section labels according to this one.

### Obiettivo

> Supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti.

Reviewer: ```suggestion
L’obiettivo della sezione `casi_d_uso` è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti. 
```

> ### Prima di iniziare

Reviewer: ```suggestion

```

already covered in the "FIle da compilare" intro

> 1. Rinomina il file in `Compilazione-EAA_[Nome Ente]_[Nome EAA].json` (es. `Compilazione-EAA_MIM_Titolo-di-studio.json`).

Reviewer: ```suggestion

```

> 2. Crea una copia del file per ciascun EAA di interesse.

Reviewer: ```suggestion

```

> ### Compilazione (sezione `sezione_casi_uso`)

Reviewer: ```suggestion
### Istruzioni di compilazione 
```

1. **metadata**: inserisci `nome_ente_titolare`, `nome_eaa` e `data_compilazione` (formato ISO: `AAAA-MM-GG`)

> 2. **risposta**: compila il campo `risposta` di ogni domanda; il campo `suggerimento` è solo indicativo

Reviewer: ```suggestion
2. **campi opzionali** e **campi obbligatori**: nel caso l'EAA si riferisce a un documento fisico o digitale già esistente (es. patente) compila sia i campi obbligatori che i campi opzionali (`volume_rilascio`, `canali_richiesta`, `formato_attuale,` `pagamento`, ecc.). Nel caso invece l'EAA non abbia un documento preesistente corrispondente, compila solo i campi obbligatori e lascia vuoti i campi opzionali. 
3. **risposta**: compila il campo `risposta` rispondendo in maniera chiara ed esaustiva ad ogni domanda; il campo `suggerimento` è solo indicativo
```

Reviewer: Moreover, how to understand if a question in the JSON file is mandatory?
How to distinguish mandatory questions (regardless of the relevance of the section "documento esistente" / "Documento non esistente") from not mandatory questions?

> 3. **campi opzionali**: i campi relativi al documento esistente (`volume_rilascio`, `limitazioni_estensioni`, `canali_richiesta`, `formato_attuale`, `pagamento`, ecc.) vanno compilati solo se l'EAA deriva da un documento già esistente

Reviewer: ```suggestion

```

Included in previous comment

**Esempio di compilazione** (fragmento):

```json
{
  "metadata": {
    "nome_ente_titolare": "Ministero dell'Istruzione",
    "nome_eaa": "Titolo di studio",
    "data_compilazione": "2026-03-16",
    "versione": "1.0"
  },
  "sezione_casi_uso": {
    "target_utenti": {
      "chi_puo_richiedere": {
        "domanda": "Chi può ad oggi richiedere il documento? ...",
        "suggerimento": "Solo maggiorenni residenti in una specifica regione",
        "risposta": "Tutti i cittadini italiani in possesso di diploma rilasciato da istituto italiano"
      }
    }
  }
}
```

> **Nota**: usa il [template form onboarding](template-form-onboarding-fonte-autentica.json). Compila tutti i campi obbligatori; i campi opzionali relativi al documento esistente vanno compilati solo se l'EAA deriva da un documento già esistente.

Reviewer: ```suggestion

```

> ### Durante la compilazione

Reviewer: ```suggestion

```

> 1. Compila il file rispondendo in maniera chiara ed esaustiva alle domande:

Reviewer: ```suggestion

```

>   - Nel caso esista ad oggi un documento, compila anche i campi specifici (volume_rilascio, canali_richiesta, formato_attuale, pagamento, ecc.)

Reviewer: ```suggestion

```

See "Istruzioni di compilazione" above

>   - Nel caso non esista ad oggi documento, compila solo i campi obbligatori e lascia vuoti quelli opzionali relativi al documento esistente

Reviewer: ```suggestion

```

> 2. Compila allo stesso modo i file delle altre EAA di interesse.

Reviewer: ```suggestion

```

> ### Validazione e checklist pre-sottomissione

Reviewer: ```suggestion

```

See "File da compilare" intro and Step 1 actions required

> **Prima di inviare il file**, è obbligatorio eseguire validazione JSON Schema e controllo sintattico. Per i comandi e il workflow, vedi [Validazione JSON Schema e Linter](validazione-json-schema-linter.md).

Reviewer: ```suggestion

```

> **Checklist**:

Reviewer: ```suggestion

```

Absorbed in Step 1 actions required

> - Validazione JSON Schema superata

Reviewer: ```suggestion

```

> - JSON Linter senza errori

Reviewer: ```suggestion

```

> - Tutti i campi obbligatori (`risposta`) compilati

Reviewer: ```suggestion

```

> - File rinominato correttamente e pronto per lo Step 2

Reviewer: ```suggestion

```

> ### Dopo la compilazione

Reviewer: ```suggestion

```
See "File da compilare" intro

> 1. Condivi i file adeguatamente rinominati e compilati, secondo le modalità previste dallo Step 2.

Reviewer: ```suggestion

```

> 2. Prosegui con le fasi successive descritte nel manuale.

Reviewer: ```suggestion

```

> 3. Mantieni sempre aggiornate le informazioni secondo le modalità definite nello Step 7.

Reviewer: ```suggestion

```

> ### Sezione casi d'uso (documento esistente o non)

Reviewer: ```suggestion

```

> Aree tematiche: **Target utenti**, **Emissione e formato**, **Utilizzo**, **Pagamento** (solo se documento esistente), **Legale e privacy**. I campi relativi al documento esistente sono opzionali.

Reviewer: ```suggestion

```

---

> ## Appendice B – Assistenza EAA

Reviewer: ```suggestion
## Appendice E – Assistenza
```

> Il modulo Assistenza EAA è incluso nel [template compilazione completo Fonte Autentica](template-form-onboarding-fonte-autentica.json), sezione `assistenza`. Ha lo scopo di supportare gli Enti nella definizione dei contenuti per l'informazione e l'assistenza all'utente relativi all'EAA.

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `assistenza` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

**Obiettivo**

L’obiettivo della sezione `assistenza` è quello di supportare gli Enti nella contribuzione al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili:
```

> L'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili:

Reviewer: ```suggestion
Nello specifico, l'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili i recapiti necessari per la risoluzione di eventuali malfunzionamenti, i canali disponibili per la gestione dell'assistenza agli utenti e i contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi da esporre all’utente in IT-Wallet.
```

> L'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili:

Reviewer: ```suggestion

```

> - contenuti utili alla predisposizione di nuove **Domande Frequenti** e/o **testi informativi** in app;

Reviewer: ```suggestion

```

> - i **recapiti necessari** per la gestione dell'assistenza agli utenti.

Reviewer: ```suggestion
```

> ### Prima di iniziare

Reviewer: ```suggestion

```

> 1. Rinomina il file compilato in `Compilazione-EAA_[Nome Ente]_[Nome EAA].json` (include anche la sezione assistenza).

Reviewer: ```suggestion
```

> 2. Crea una copia del file per ciascun EAA di interesse.

Reviewer: ```suggestion

```

> ### Contenuti da compilare (sezione `assistenza`)

Reviewer: ```suggestion
### Istruzioni di compilazione
```

> 1. **referenti** — Referente ambito sistemistico, Referente ambito applicativo, Referente ambito assistenza (nome, cognome, email, telefono)

Reviewer: ```suggestion
1. **Referenti** — Fornisci e mantieni aggiornati, i dati di contatto (nome, cognome, email, telefono) di almeno 1 **referente per l’assistenza agli utenti**, 1 **referente in ambito applicativo** e 1 **referente in ambito sistemistico** che possano prontamente collaborare con il Fornitore di Attestati Elettronici di Attributi (IPZS per gli Attestati Elettronici di interesse pubblico) e con il Fornitore di Wallet (PagoPA per la soluzione IT-Wallet pubblica) per la risoluzione congiunta di segnalazioni degli utenti e/o malfunzionamenti tra i servizi.
```

> 2. **canali** — Email assistenza, Numero telefonico, Altro

Reviewer: ```suggestion
2. **Canali** — Fornisci almeno un canale di assistenza (es. e-mail assistenza, numero di telefono, etc.) di responsabilità dell’Ente che rappresenti, nel ruolo di Titolare di Fonte Autentica, a cui si possa indirizzare l’utente per quelle richieste di supporto e segnalazioni non gestibili all’interno della soluzione IT-Wallet. 
```

> 3. **faq** — Domande Frequenti e relative risposte

Reviewer: ```suggestion
3. **FAQ** — Contribuisci alla definizione dei contenuti utili a predisporre le domande più frequenti e comuni relative all’EAA in analisi, dall’emissione all’utilizzo fino alla gestione del suo ciclo vita.
```

> 4. **testi_informativi** — Testi o messaggi da esporre agli utenti nella soluzione IT-Wallet

Reviewer: ```suggestion
4. **testi_informativi** (opzionale) — Se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 
- Poniti le seguenti domande (puoi fare riferimento alla sezione “casi_d_uso”, già compilato):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
- Formula un testo informativo rivolto all’utente a partire dai contenuti sopra raccolti, assicurandoti che sia chiaro, semplice, diretto e conciso (circa 300 - 450 caratteri, spazi inclusi).
```

> ### Dopo la compilazione

Reviewer: ```suggestion

```

> 1. Condivi il file adeguatamente rinominato e compilato, secondo le modalità previste dallo Step 2.

Reviewer: ```suggestion

```

> 2. Prosegui con le fasi successive descritte nel manuale.

Reviewer: ```suggestion

```

> 3. Mantieni sempre aggiornate le informazioni nella sezione `assistenza` secondo le modalità definite nello Step 7.

Reviewer: ```suggestion

```

| CODICE  | ESITO                                                                | APPLICABILE | CAUSA                                          | AZIONE UTENTE                        | NOTE |
| ------- | -------------------------------------------------------------------- | ----------- | ---------------------------------------------- | ------------------------------------ | ---- |
| 200     | Attestato digitale valido                                            | SI          | Dati ritornati correttamente                   | N/A                                  |      |
| 400-451 | Errori client (Bad Request, Unauthorized, ecc.)                      |             |                                                |                                      |      |
| 404     | Not found                                                            | SI          | Non sono stati trovati documenti di titolarità | Chiudere e riprovare successivamente |      |
| 500-503 | Errori server (generico, servizio non implementato, non disponibile) |             |                                                |                                      |      |

> | 540     | EAA non esistente presso l'Authentic Source                          |             | Formato `"state": "description"`               |                                      |      |

Reviewer 2: @peppelinux HTTP status codes 540 and 541 are not currently supported by the YAML defined for ASs in the IT-Wallet guidelines. Which one should be used?

Author: I am completely neutral with that, I'd leave it up to the review of ipzs @m-basili @pcerqua-ipzs @fmarino-ipzs ^

Author: or PDND team in pagopa, unfortunately I don have them in this project yet, therefore no way to tag them for review

Reviewer 2: Solved with @RosaliaGaleano and @ChiaLeon, HTTP status that are not in the specs are not mandatory

Il presente manuale è da intendersi anche come uno strumento aperto alla condivisione e alla collaborazione tra gli Enti interessati. 

## Ruoli e responsabilità del Titolare di Fonte Autentica

Il Titolare di Fonte Autentica (Authentic Source - AS) è il soggetto che, nel Sistema IT-Wallet, detiene i dati da cui vengono creati gli Attestati Elettronici di Attributi rilasciati nelle soluzioni IT-Wallet. Ad esempio, per la Patente di Guida il Titolare di Fonte Autentica è la Direzione Generale di Motorizzazione del Ministero delle Infrastrutture e dei Trasporti. 

L'Ente che fornisce i dati, in quanto Titolare di Fonte Autentica, è l'unico soggetto che può definire mezzi e finalità di uso degli stessi. L'Ente rimarrà sempre il "proprietario" del dato e sarà responsabile del ciclo di vita del dato quindi di eventuali modifiche o cambiamenti di stato. Il Sistema IT-Wallet è concepito in modo che l'Ente possa continuare a gestire il dato in modo autonomo e conforme alle proprie politiche e alle normative vigenti. 

Di seguito rappresentato il ruolo del Titolare di Fonte Autentica e delle altre entità nel Sistema IT-Wallet. 

> Figura 1

Author: acc, mancano le figure!

Author: fixed e191e9a7bafafdcf3e449513fcbdd2b8a8f6cc3b
